seeds/3_files.js

```diff
@@ -3,46 +3,48 @@
 var fs = require('fs');
 
 exports.seed = function(knex, Promise) {
+  // As knex does not have an .insertIfNotExists(), we hook into catch()
+  // so that seeding doesn't break when running on an already seeded database.
   return Promise.join(
     knex('files').insert({
       project_id: 1,
       path: '/spacecats-API/index.js',
       buffer: fs.readFileSync(__dirname + '/file_data/index.js')
-    }),
+    }).catch(function() { return knex('files'); }),
```

Comment:
Seeding the DB used by the service is valuable in development. What's stopping us from having a solution that allows both?